### PR TITLE
Add initial issue and PR templates

### PR DESCRIPTION
**Pull request by austinlparker**
Hopefully this can direct people to the appropriate place to file issues/PRs for docs...

### CODE REVIEW

**Comment by mxiamxia** [comment]
Can we update it in https://github.com/open-telemetry/opentelemetry.io/blob/main/CONTRIBUTING.md?

**Comment by austinlparker** [comment]
> Can we update it in https://github.com/open-telemetry/opentelemetry.io/blob/main/CONTRIBUTING.md?

It's already in there.

File: .github/ISSUE_TEMPLATE/DOCS_UPDATE.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+labels: bug
+assignees: ''
+
+---
+
+## IMPORTANT - READ THIS FIRST
+The authoritative copy of the documentation for each language is contained in the respective language repository at `/website_docs`.
+Please report any issues in the appropriate SIG repository.
+
+**What needs changed?**
```
Comment (mtwo) [inline .github/ISSUE_TEMPLATE/DOCS_UPDATE.md:14]:
```suggestion
**What needs to be changed?**
```

**Review by mtwo — APPROVED** [review]
One change needed, looks great otherwise

**Pull request merged**